### DISCUSSION TITLE
Practicas Y Actividades Launch X

### DISCUSSION THREAD

**Discussion by Gilberto-Guzman** [Launch-X-Latam/MisionFrontEnd#335] [Show and tell]
## Hola, Si estas interesado en visualizar mi progreso de las practicas de Launch-X-Latam, accede al siguiente enlace, este te enviara a mi cuenta principal: https://github.com/Gilberto-Guzman

![cat-typing](https://user-images.githubusercontent.com/92352192/173468500-01a8cf34-74c6-47ab-965e-edde98672415.gif)

### Adicionalmente me gustaría conocer su progreso, para conocer diversos puntos de vista para la resolución de problemas :D

**Comment by Gil987654321** [comment]
Además, próximamente tendré un Backup, en mi cuenta secundaria. 😎
